.gitlab-ci.yml

```diff
@@ -450,3 +450,127 @@ publish:mongodb-backup-runner:
   script:
     - *tag_n_push_to_gitlab_registry
 
+.template:trigger:acceptance-testing:
+  stage: test
+  needs:
+    - job: build:acceptance-testing
+      artifacts: false
+  inherit:
+    variables: false
+  variables:
+    CONTAINER_TAG: "acceptance-testing"
+    MENDER_TEST_CONTAINERS_CANDIDATE_TAG: ${CI_REGISTRY_IMAGE}:${CONTAINER_TAG}-${CI_PIPELINE_ID}
+
+.template:trigger:backend-integration-testing:
```

Comment:
This template is only used once, I think is better to have in on a single job and avoid confusion (it is not design to be reused for more job)

Reply:
fixed.